assets/sass/components/global/_googlesitekit-header.scss

```
@@ -117,6 +117,19 @@
 				padding-right: $grid-gap-desktop;
 			}
 
+
+			.googlesitekit-header__date-range-selector-menu--has-unified-dashboard {
+
+				&.mdc-button--dropdown {
+
+					@media (max-width: $bp-mobileOnly) {
+
+						background: none;
+						padding: 8px;
```

I think it should be `padding-right`, shouldn't it?
```suggestion
						padding-right: 8px;
```

---

Thanks for spotting that - I had intended to specify `padding-right` here, somehow this slipped past me. That's now updated.